internal/provider/config.go

```diff
@@ -118,7 +119,7 @@ func (c *Config) NewClient() (*client.Client, error) {
 	}
 
 	// If there is no cert information, then check for ssh://
-	helper, err := connhelper.GetConnectionHelper(c.Host)
+	helper, err := connhelper.GetConnectionHelperWithSSHOpts(c.Host, c.SSHOpts)
```

Comment:
👍 Even if SSHOpts is nil, there is no problem.
https://github.com/docker/cli/blob/v20.10.12/cli/connhelper/connhelper.go#L20-L35

Reply:
yep :+1: